### PR TITLE
Followup revisions for Sample keyword search fixes:

### PR DESCRIPTION
**Pull request by Luke-Sikina**
Fix #6746
Describe changes proposed in this pull request:
- Formatting
- Two new db tests
    - Verify that samples are filtered by study
    - Verify that an empty list of studies results in no samples

# Checks
- [ ] Runs on heroku
- [x] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.

### CODE REVIEW

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepositoryTest.java [hunk 1]
```diff
@@ -346,6 +347,32 @@ public void getSamplesByKeyword() {
 
          Assert.assertEquals(expected, actual);
      }
+    
+     @Test
```
Comment (sheridancbio) [inline persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepositoryTest.java]: Here the @Test annotation is indented one space too far

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepositoryTest.java [hunk 2]
```diff
@@ -346,6 +347,32 @@ public void getSamplesByKeyword() {
...
+                "TCGA-A1-A0SB-01",
+                "TCGA-A1-B0SO-01",
+                "TCGA-A1-B0SP-01",
+                "TCGA-A1-B0SQ-01");
+        
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getSamplesByKeywordFilterByNoStudies() {
```
Comment (sheridancbio) [inline persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepositoryTest.java]: 👍

File: persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepository.java [hunk 3]
```diff
@@ -21,13 +21,12 @@
...
-        String keyword, List<String> studyIds, String projection,
-        Integer pageSize, Integer pageNumber, String sort, String direction
+            String keyword, List<String> studyIds, String projection,
+            Integer pageSize, Integer pageNumber, String sort, String direction
     ) {
         return sampleMapper.getSamples(
-            studyIds, null, null, keyword, projection, pageSize,
-            offsetCalculator.calculate(pageSize, pageNumber), sort, direction
-        );
+                studyIds, null, null, keyword, projection, pageSize,
```
[the remark this answers is not in the document]
Comment (inodb) [reply persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepository.java]: Might be worth finding a Java code formatter for this and let CI handle these things. E.g. for frontend we are using "prettier". I remember seeing something in this project when I made edits there: https://github.com/bdemers/maven-external-version

[the remark this answers is not in the document]
Comment (inodb) [reply persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/SampleMyBatisRepository.java]: Opened a separate issue for this here: https://github.com/cBioPortal/cbioportal/issues/6961

**Comment by inodb** [comment]
Gonna merge this because it's only adding tests - it's a little tricky to read the PR due to the many formatting changes. Should really only be changes to: SampleMyBatisRepositoryTest.java. Let's follow up with https://github.com/cBioPortal/cbioportal/issues/6961

**Pull request merged**